qt_ui/windows/mission/flight/QFlightCreator.py

```diff
@@ -32,6 +32,7 @@
 
 class QFlightCreator(QDialog):
     created = Signal(Flight)
+    pilots_changed = Signal()
```

Comment:
I believe it would be a better idea to drop this down to ~~`FlightRosterEditor`~~ `QFlightSlotEditor` (I believe that's the class that's furthest "down the rabbit hole" if I haven't missed anything), and then use that signal from `self.roster_editor.pilots_changed` instead, that way we don't create a dependency on a Signal object in `FlightRosterEditor`'s constructor. Does that make sense? Or have I missed something? 😅

Reply:
Done, I think. Let me know if I missed the old pilots_changed signal somewhere?

Reply:
Looks good 👍